shiipou/simplon-2024-brief-06-TooManyChoco
# 33 gestion des logins api

## About this PR



## Technical highlight/advice


## How to test my changes

1- .\gradlew run
2- using Thunder Client extension or PostMan go to localhost/8080/login and in the body put {
  "email": "bobo@maif.fr",
  "password": "bobo123"
}
make sure you're using post method
You should recive back a json object like this one 
{
  "username": "bob",
  "firstName": "bobo",
  "email": "bobo@maif.fr",
  "password": "bobo123"
}
## Checklist

### I included unit tests that cover my changes

### I added/updated the documentation about my changes
